src/duckstation-qt/isobrowserwindow.ui

```diff
@@ -133,6 +133,9 @@
        <property name="text">
         <string>Close</string>
```

Comment:
btw, a close button for the ISO browser feels unnecessary IMHO. The same goes for the cover downloader. Both are "tools" windows that feel somewhat independent from the main UI. The memcard editor, which is also standalone, already lacks a close button.

I may be influenced by OS-specific UI paradigms, as close (or OK) buttons are uncommon on macOS, except on modal dialogs. Maybe it's different on Windows...

Reply:
Yeah, I think there is a difference between the Windows and MacOS HIG here - as far as I'm aware, most dialogs on Windows will have some sort of action button that closes the dialog. Whether that's OK/Cancel, Close, etc.

Guess we could hide it on MacOS, but for the settings window the restore defaults button would be pretty lonely down there without a friend.